touch-events/historical.html

```diff
@@ -32,15 +32,6 @@
                "Should not be supported on the instance");
 }, "TouchList::identifiedTouch");
 
-test(function() {
-  assert_false("initTouchEvent" in TouchEvent.prototype,
-               "Should not be supported on the prototype");
-
-  var touchEvent = document.createEvent("touchevent");
-  assert_false("initTouchEvent" in touchEvent,
-               "Should not be supported on the instance");
```

Comment:
Keep this test, but use the constructor to create `touchEvent` instead.

Reply:
Done, thanks!